src/uuid.cr

```diff
@@ -164,6 +164,11 @@ struct UUID
     to_slice == other.to_slice
   end
 
+  # Convert to `String` in literal format.
+  def inspect(io : IO)
+    to_s(io)
```

Comment:
One thing about inspect is that it typically gives some hint of what type of object it is - inspect is for debugging purposes after all.  

I'm not certain how it normallt looks, but I wouldn't mind something like `UUID[#{to_s}}`, that is, both showing that it is a UUID and showing the string.

Reply:
```cr
io << "UUID("
to_s(io)
io << ')'
```

Reply:
uh, please stick to

```cr
io << "#<Klass:"
to_s(io)
io << '>'
io
```

convention. It's been used throughout stdlib.

Reply:
I'm not sure we really need that. UUID is not one of the basic types, but the presentation format is pretty significant to identify.

Reply:
`Type(..)` is the standard for structs but for references it's `#<Type ...>`

Yeah weird I know.

Reply:
@RX14 I did. `File::Stat`, `XML::Namespace` or `XML::Node` for example prove sth different.

Reply:
well I guess we need a standard then.

For now, thios is fine.